pkgs/os-specific/linux/systemd/default.nix

```diff
@@ -87,6 +88,12 @@
   bpftools,
   libbpf,
 
+  # vmlinux.h file containing the kernel's BTF information,
+  # used throughout systemd's BPF programs.
+  # When building systemd for a customized kernel, you'll want
```

Comment:
Would I want to do that? I think no, these are basically `linuxHeaders`. While it might make sense to make this overridable, I think the comment makes the weird suggestion that `vmlinux.h` needs to be "synchronized" with the kernel which is just not necessary.